### ISSUE TITLE
Failed to resolve '../patterns/en-us.cjs.js' since 1.10.2

### ISSUE THREAD

**Issue by ebo-zig** [ytiurin/hyphen#94]
Since release 1.10.2 we are facing the following error:

`Module not found: Error: Can't resolve '../patterns/en-us.cjs.js' in '/data/node_modules/hyphen/en-us.cjs'`

It looks like the `patterns/en-us.cjs` file is exported to NPM but is not present in the patterns folder:
https://www.npmjs.com/package/hyphen?activeTab=code

The code in which we are using `hyphen` is as follows:

```
const { hyphenate } = await import(`hyphen/${language}`);
const hyphenatedText: string = await hyphenate(text, { hyphenChar: '&shy', minWordLength: 10 })
    .then(result => {
        return result;
    })
    .catch(() => {
        return text;
    });
```

Maybe the way we dynamically import `hyphen` is causing an issue?

**Comment by ytiurin** [comment]
Hi, this happend because test file slipped into the build.
This bug is now fixed in version 1.10.4

**Comment by ebo-zig** [comment]
Thanks for the quick solution!
It's working with 1.10.4
